main.go

```diff
@@ -387,9 +388,18 @@ func printableProfileInfos(profileContent string) (string, error) {
 		if isProvisionedDevicesSection {
 			if strings.Contains(line, provisionedDevicesArrayEndLine) {
 				isProvisionedDevicesSection = false
-				lines = append(lines, fmt.Sprintf("%s[REDACTED]", strings.Repeat(" ", 16)))
+				lines = append(lines, line)
+			} else if !strings.Contains(line, provisionedDevicesArrayStartLine) {
+				deviceID := strings.TrimSpace(strings.NewReplacer("<string>", "", "</string>", "").Replace(line))
+				deviceIDSubset := ""
```

Comment:
we do not use `deviceIDSubset ` in this scope, please declare this in the scope, where you use it!